libs/core/src/lib/multi-input/multi-input-mobile/multi-input-mobile.component.ts

```diff
@@ -0,0 +1,152 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    isDevMode,
+    OnDestroy,
+    OnInit,
+    TemplateRef,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { DialogService } from '../../dialog/dialog-service/dialog.service';
+import { DialogRef } from '../../dialog/dialog-utils/dialog-ref.class';
+import { DropdownMobileConfiguration } from './dropdown-mobile-configuration';
+import { MultiInputComponent } from '../multi-input.component';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+@Component({
+    selector: 'fd-multi-input-mobile',
+    templateUrl: './multi-input-mobile.component.html',
+    styleUrls: ['./multi-input-mobile.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None
+})
+export class MultiInputMobileComponent implements OnInit, AfterViewInit, OnDestroy {
+
+    /** @hidden */
+    multiInputConfig: DropdownMobileConfiguration;
+
+    /** @hidden
+     * For internal usage
+     * Control element, which will be rendered inside dialog.
+     * List element, which will be rendered inside dialog.
+     */
+    childContent: {
+        listTemplate: TemplateRef<any>,
+        controlTemplate: TemplateRef<any>
+    } = null;
+
+    /** @hidden */
+    @ViewChild('dialogTemplate') dialogTemplate: TemplateRef<any>;
+
+    /** @hidden */
+    private _dialogRef: DialogRef;
+
+    /** @hidden */
+    private _selectedBackup: any[];
+
+    /** An RxJS Subject that will kill the data stream upon component’s destruction (for unsubscribing)  */
+    private readonly _onDestroy$: Subject<void> = new Subject<void>();
+
+    constructor(
+        private _dialogService: DialogService,
+        private _multiInputComponent: MultiInputComponent,
+        private _changeDetRef: ChangeDetectorRef,
+        private _elementRef: ElementRef
+    ) {}
+
+    /** @hidden */
+    ngOnInit(): void {
+        this.multiInputConfig = this.getMultiInputConfig();
+        if (this.multiInputConfig) {
+            this._listenOnMultiInputOpenChange();
+        }
+    }
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._open();
+        this._dialogRef.hide(true);
+    }
+
+    /** @hidden */
+    ngOnDestroy(): void {
+        this._onDestroy$.next();
+        this._onDestroy$.complete();
+    }
+
+    /**
+     * Method returns, if the dialog is open.
+     * It gives information to prevent from having more than 1 dialog .
+     * Also removes from DOM, <input> control element, which is rendered in dialog.
+     */
+    hasOpenDialogs(): boolean {
+        return this._dialogService && this._dialogService.hasOpenDialogs();
```

Comment:
`return this._dialogService?.hasOpenDialogs()`

Reply:
Since we have mandatory dialogService, this method is useless.